src/ngraph/builder/quantization_util.hpp

```diff
@@ -99,6 +99,55 @@ namespace ngraph
                     (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
                 return scale;
             }
+
+            template <typename T>
+            static inline T get_quantization_scale(const std::shared_ptr<Node> min_input,
+                                                   const std::shared_ptr<Node> max_input,
+                                                   const ngraph::element::Type& type,
+                                                   bool bump_by_eps = false)
+            {
+                auto min_input_const_op =
+                    std::dynamic_pointer_cast<ngraph::op::Constant>(min_input);
+                auto max_input_const_op =
+                    std::dynamic_pointer_cast<ngraph::op::Constant>(max_input);
+
+                if (min_input_const_op == nullptr)
+                    throw ngraph_error("min input must be constant");
+                else if (max_input_const_op == nullptr)
+                    throw ngraph_error("max input must be constant");
```

Comment:
The const assumption here breaks the functionality of the general quantize/dequantize ops in both TF and MXNet. I cannot implement these ops in the bridge using this builder as it stands and still pass unit tests. Thus, I approve under strong objection mostly to move the other projects forward.